src/System.Console/src/System/Console.cs

```diff
@@ -728,31 +728,34 @@ public static void Write(String value)
             Out.Write(value);
         }
 
-        // TODO: Uncomment when ArgIterator and friends are available in System.Runtime.dll
```

Comment:
These aren't tests, this are the actual implementation for the Console.WriteLine and Console.Write apis that are being exposed on this PR as they where added to N.S 2.0 as well. Did you mean other tests? @danmosemsft